src/promptflow-tracing/promptflow/tracing/_start_trace.py

```diff
@@ -44,6 +44,11 @@ def start_trace(
         logging.info("skip tracing local setup as the environment variable is set.")
         return
 
+    # openai instrumentation
+    logging.debug("injecting OpenAI API...")
+    inject_openai_api()
```

Comment:
Why we need this code?

Reply:
this is missed during the storage refactor, we shall instrument OpenAI in `start_trace`, just add in this PR to avoid a separate small PR as we have tight time for bug bash...